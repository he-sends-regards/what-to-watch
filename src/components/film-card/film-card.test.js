import React from "react";
import renderer from "react-test-renderer";
import FilmCard from "./film-card";
import mockPromoFilm from "../../mocks/promoFilm";
import browserHistory from "../../browser-history";
import {Router} from "react-router-dom";

const noop = () => {};

it(`Should FilmCard render correctly`, () => {
  expect(renderer.create(
      <Router history={browserHistory}>
        <FilmCard
          onMouseEnter={noop}
          onMouseLeave={noop}
          isActive={false}
          film={mockPromoFilm}
        />
      </Router>
  ).toJSON()).toMatchSnapshot();
});
