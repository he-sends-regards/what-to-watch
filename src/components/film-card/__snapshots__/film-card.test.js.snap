// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FilmCard render correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  data-id={1}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    href="/films/1"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Bronson"
        height="175"
        src="https://assets.htmlacademy.ru/intensives/javascript-3/film/preview/bronson.jpg"
        width="280"
      />
    </div>
  </a>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films/1"
      onClick={[Function]}
    >
      Bronson
    </a>
  </h3>
</article>
`;
